src/applications/vaos/containers/ReviewPage.jsx

```diff
@@ -86,7 +86,7 @@ export class ReviewPage extends React.Component {
             status="error"
             headline={
               submitStatusVaos400
-                ? 'We can’t schedule your appointment'
+                ? "We're sorry."
```

Comment:
We try to use the apostrophe character in user-facing content: `’` instead of `'`

Also, the way I read the requested change, I think we should be putting "We’re sorry. " in front of both of the headlines here.

Reply:
Pushing fix now